UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Common.UnitTests/HealthCheck/FileShareServiceHealthCheckTest.cs

```diff
@@ -0,0 +1,47 @@
+﻿using FakeItEasy;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Microsoft.Extensions.Logging;
+using NUnit.Framework;
+using System;
+using System.Threading.Tasks;
+using UKHO.MaritimeSafetyInformation.Common.HealthCheck;
+
+namespace UKHO.MaritimeSafetyInformation.Common.UnitTests.HealthCheck
+{
+    [TestFixture]
+    public class FileShareServiceHealthCheckTest
+    {
+        private ILogger<FileShareServiceHealthCheck> _fakeLogger;
+        private IFileShareServiceHealthClient _fakeFileShareServiceHealthClient;
+        private FileShareServiceHealthCheck _fileShareServiceHealthCheck;
+
+        [SetUp]
+        public void Setup()
+        {
+            _fakeLogger = A.Fake<ILogger<FileShareServiceHealthCheck>>();
+            _fakeFileShareServiceHealthClient = A.Fake<IFileShareServiceHealthClient>();
+
+            _fileShareServiceHealthCheck = new FileShareServiceHealthCheck(_fakeFileShareServiceHealthClient, _fakeLogger);
+        }
+
+        [Test]
+        public async Task WhenFssConnectionIsHealthy_ThenReturnsHealthy()
```

Comment:
Refer Fss as FSS please.
```suggestion
        public async Task WhenFSSConnectionIsHealthy_ThenReturnsHealthy()
```

Reply:
changes done.